### PR TITLE
Pawelka/aspnetcore

### PR DESCRIPTION
**Pull request by moozzyk**
Issue #95 (point 2, part 1)

### CODE REVIEW

**Comment by Tratcher** [comment]
So this is a straight copy and rename of both the tool and tests? I assume you're planning to merge it at leave the old tool in place for a while?

You missed updating the SLN file and the nugetpackageverifier.json.

**Comment by moozzyk** [comment]
Yes. 1st commit is a copy. 2nd commit is changing to aspnetcoremodule. 3rd commit is reacting to changes in env variables. dotnet-publish-iis stays as is for now. Once aspnetcoremodule ships we will be able to remove it. 
Good point about sln and package verifier. I will update both.

File: test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        public void PublishIIS_modifies_existing_web_config()
+        {
+            var webRoot = "wwwroot";
+            var folders = CreateTestDir("{}", webRoot);
+
+            File.WriteAllText(Path.Combine(folders.PublishOutput, webRoot, "web.config"),
+@"<configuration>
+  <system.webServer>
+    <handlers>
+      <add name=""aspNetCore"" path=""*"" verb=""*"" modules=""aspNetCoreModule"" resourceType=""Unspecified""/>
```
Comment (Tratcher) [inline test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs]: FYI: the web.config is case sensitive, this needs to say `AspNetCoreModule`.

Comment (moozzyk) [inline test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs]: Good catch. I must have broken it with one of the bulk replaces.. Will fix.

**Comment by muratg** [comment]
You'll also need to update the `packages.csv` file. This is used by the split-packages tool (categorizing packages to shipping/non-shipping/external).

File is here: https://github.com/aspnet/specs/blob/master/packages/packages.csv

**Comment by moozzyk** [comment]
:up::date:

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**